src/components/BankIconButton.vue

```diff
@@ -1,26 +1,30 @@
 <template>
     <button class="reset bank-icon-button flex-column" v-on="$listeners">
         <BankIcon/>
+        <TriangleDownIcon class="triangle"/>
```

Comment:
I would prefer adding the class `triangle-down-icon` to the `TriangleDownIcon` component and then using that class.